adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/generator/RestAdapterGenerator.java

```diff
@@ -19,30 +21,31 @@
 import static org.apache.commons.lang.Validate.isTrue;
 import static org.apache.commons.lang.Validate.notEmpty;
 import static org.apache.commons.lang.Validate.notNull;
+import static uk.gov.justice.services.adapters.rest.generator.Names.componentFromBaseUriIn;
 
 public class RestAdapterGenerator implements Generator {
 
     private final RamlValidator validator = new CompositeRamlValidator(
```

Comment:
If this just instantiates new instances shouldn't this just be done constructor side in a default constructor?